src/main/typesystem/SearchResultTypes_TS.xml

```diff
@@ -30,6 +30,11 @@
             <description>ID of the document (specific to data source)</description>
             <rangeTypeName>uima.cas.String</rangeTypeName>
           </featureDescription>
+            <featureDescription>
+                <name>sourceID</name>
+                <description>ID of source</description>
```

Comment:
This seems completely non-descriptive to me, someone who doesn't know the dashboard terminology is going to be totally confused.